llvm/lib/Transforms/Vectorize/VPlanHCFGBuilder.cpp

```diff
@@ -156,6 +156,19 @@ static bool isHeaderVPBB(VPBasicBlock *VPBB) {
   return VPBB->getParent() && VPBB->getParent()->getEntry() == VPBB;
 }
 
+/// Return true of \p L loop is contained within \p OuterLoop
+static bool isNestedLoop(const Loop *L, const Loop *OuterLoop) {
```

Comment:
nit: might be better to call it something like `doesContainLoop`, more in line with the comment.